ncm-authconfig/src/main/pan/components/authconfig/sssd.pan

```diff
@@ -117,7 +117,7 @@ type authconfig_sssd_domain  = {
     "max_id" : long = 0
     "enumerate" : boolean = false
     "timeout" : long = 10
-    "force_timeout" : long = 60
+    "force_timeout" ? long with {deprecated(0, "Unsupported in later versions"); true}
```

Comment:
This results in a context-less warning message which does not mention the path. Can you please change it to say something like, “Warning: sssd/force_timeout option is unsupported in later releases of sssd and will be removed from the next release of Quattor”